README.md

```diff
@@ -20,12 +20,12 @@ There are several ways to run CTF through gemelli. Here we provide a tutorial of
 
 ### Tutorials with QIIME2
 
-* [QIIME2 Command Line Interface](https://github.com/cameronmartino/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md)
-* [QIIME2 API](https://nbviewer.jupyter.org/github.com/cameronmartino/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-API.ipynb)
+* [QIIME2 Command Line Interface](https://github.com/biocore/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md)
```

Comment:
I think there are a number of other links like this in the jupyter notebooks that should be changed as well, e.g.,

https://github.com/cameronmartino/gemelli/blob/825beea36eeda4e7e9b0bd988c1b21c5d26f6e2a/ipynb/tutorials/IBD-Tutorial-QIIME2-API.ipynb#L18-L22

Reply:
Good catch - thanks!